### ISSUE TITLE
Can only see the Game and App folders on Simplemenu

### ISSUE THREAD

**Issue by nadirkhansa** [fgl82/simplemenu#102]
Hi,

Thanks in advance for your help. 
I have followed MashTec's instructions for installation as well as reviewing the comments and readme on Git and while I am able to launch the app, I can only see the Applications and Game tabs.
I have an RG350M (Stock Firmware), with ROMS installed on an SD card (named RG350) and Emulators on the internal card (media/data/apps)

Here is what I have already tried:

1. Deleted /media/data/local/home/.simplemenu/last_state.sav
2. In [/media/data/local/home/.simplemenu/section_groups], I have double checked and confirmed section files for arcade.ini + consoles.ini + handhelds.ini + home computers.ini have the following changes;
2.1 consolelist = removed unused consoles
2.2 romDirs have updated paths to /media/RG350/apps/xxxx.opk
2.3 Emulators OPK names in /media/data/apps have the correct names (according to the config files)

Not sure what I might be missing here, but appreciate the assistance.
I have also attached the log file, which looks like it goes into a loop when loading the APP menu and then encounters a Critical Error.

[2020-06-22.log](https://github.com/fgl82/simplemenu/files/4810694/2020-06-22.log)

**Comment by thewookiemaster** [comment]
Hello,
I am having the same issue as nadirkhansa and followed the exact same steps.
Thank you

**Comment by nadirkhansa** [comment]
Hi, Update: 23/06/2020 (still having the issue)

Device: RG350M
Firmware: Stock Firmware - Build date: May 10 2020
SM OPK: media/RG350/APPS (on the External sdcard)
ROM folder: media/RG350/ROMS/ (on External sd card)
Emulators folder: media/data/apps (on Internal sd card)

Here is what I have now done.
1. I cleaned up the previous installation, removed the OPK and the .Simplemenu folder.
2. Opened SM and closed it
3. Deleted the lastsave file
The next steps, I changed the configuration to help with troubleshooting, to minimise what was running/loading after SM started.
4. Configured the ini files in the section group (see changes below);

> 4.1 **apps and games.ini**: I have only enabled the GAMES screen - see file attached.
> _[apps and games.ini.txt](https://github.com/fgl82/simplemenu/files/4822367/apps.and.games.ini.txt)_
> 
> 4.2 **arcades.ini**:  only enabled FINALBURN ALPHA and verified the path to the apps and roms directory is correct. My sdcard is "RG350".
> _[arcades.ini.txt](https://github.com/fgl82/simplemenu/files/4822385/arcades.ini.txt)_
> 
> 4.3 **consoles.ini**: removed all consoles from the list, so that none will run
> _[consoles.ini.txt](https://github.com/fgl82/simplemenu/files/4822386/consoles.ini.txt)_
> 
> 4.4 **handhelds.ini**: removed all consoles from the list, so that none will run
> _[handhelds.ini.txt](https://github.com/fgl82/simplemenu/files/4822388/handhelds.ini.txt)_
> 
> 4.5 **home computers.ini**: removed all consoles from the list, so that none will run
> _[home computers.ini.txt](https://github.com/fgl82/simplemenu/files/4822389/home.computers.ini.txt)_
> 
5. The aim of this was to simply start SM, with one emulator active then close it and look at the log file.
6. Started SM, press start, select quit. - note at this point I have not pressed any other buttons or tried to switch screens.
7. Attached log file for your review., from what I can see, simply starting the app with the minimal configuration gives **ERROR | Nice, a critical error!!!**
[2020-06-23.log](https://github.com/fgl82/simplemenu/files/4822351/2020-06-23.log)

PS: after clearing the logs, I then started SM again and try to and switch to Arcade, but it only displayed the Game page... so simply starting SM and exiting generates an error and starting SM, trying to switch to the next section (Arcades) also generates an error and I don't see the emulators.

Hope this helps!

**Comment by nadirkhansa** [comment]
Quick update...
When looking at the previous log, it appeared, that the error came up "after" the Game screen was loaded.
So I updated all the section_group config files, so that none of the consoleList was left black. So there was a console specified and a corresponding OPK and rom folder existed.

I then opened SM and closed it again (did not do anything else) and this seems to have gotten rid of the Critical error.
When I looked at the log again (see attached). There is no critical error and these are the last few entries:
Wed Jun 24 00:37:44 2020 | INFO  | Scanning directory
Wed Jun 24 00:37:44 2020 | INFO  | Processing files
Wed Jun 24 00:37:44 2020 | INFO  | All files loaded
Which looks like it loads the emulators, scans the directories and loads files.... but no error this time.
[2020-06-24.log](https://github.com/fgl82/simplemenu/files/4822438/2020-06-24.log)
